recipes/mecab-python3/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mecab-python3" %}
+{% set version = "1.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mecab-python3-{{ version }}.tar.gz
+  sha256: 62abe28a1155398325372291483608427bc82681fef80e7d132904415f9fd42e
+
+build:
+  number: 0
+  entry_points:
+    - mecab-py = MeCab.cli:parse
+    - mecab-py-info = MeCab.cli:info
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools_scm
+    - mecab
+  run:
+    - python
+    - mecab
```

Comment:
B/c this recipe is added here too there will be a race condition when we merge this PR and `mecab-python3` will fail the first run. Bare in mind that you'll have to restart the CIs for it later.

Reply:
Noted.  Thanks for the heads-up.